packages/core/src/extensions/DraggableBlocks/DraggableBlocksPlugin.ts

```diff
@@ -184,8 +182,10 @@ function dragStart(e: DragEvent, view: EditorView) {
     return;
   }
 
+  const editorBoundingBox = view.dom.getBoundingClientRect();
```

Comment:
fyi, this call to getBoundingClientRect _might_ be expensive (because it forces a reflow), but I couldn't see any performance issues using chrome inspector. I was thinking we might need some smart caching, but let's keep it as-is until we really see there's a bottleneck